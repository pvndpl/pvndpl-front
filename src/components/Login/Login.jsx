import style from "./Login.module.css";
import './form.css'
const Login = () => {
    return (
        <div className={style.vkloginHeader}>

            <p className={style.vkloginHeaderPretitle}>Добро пожаловать в</p>
            <p className={style.vkloginHeaderTitle}>ПИВАНДОПАЛУ</p>

            <div className={style.loginForm}>
                <div className="container">
                    <div className="screen">
                        <div className="screen__content">
                        <h1>Заходи</h1>
                            <form className="login">
                                <div className="login__field">
                                    <input type="text" className="login__input" placeholder="User name" />
                                </div>
                                <div className="login__field">
                                    <input type="password" className="login__input" placeholder="Пароль" />
                                </div>
                                <button className="button login__submit">
                                    <span className="button__text">Зайти</span>
                                </button>
                            </form>
                        </div>
                        <div className="screen__background">
                            <span className="screen__background__shape screen__background__shape4"></span>
                            <span className="screen__background__shape screen__background__shape3"></span>
                            <span className="screen__background__shape screen__background__shape2"></span>
                            <span className="screen__background__shape screen__background__shape1"></span>
                        </div>
                    </div>
                </div>

            </div>
        </div>
    );
}

export default Login