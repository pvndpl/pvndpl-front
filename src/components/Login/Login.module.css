.vkloginHeaderTitle {
    font-size: 6.5rem;
    font-weight: 900;
    color: white;
}
.vkloginHeaderPretitle {
    font-size: 1.5rem;
    font-weight: 500;
    color: white;
}

.vkloginHeader{
    width: 100%;
    height: 100%;
    background: url('https://odindesignthemes.com/vikinger-theme/wp-content/themes/vikinger/img/login/background.jpg') no-repeat 0 0, right 40px top 40px no-repeat, url("https://odindesignthemes.com/vikinger/img/landing/dot-texture.png") repeat 0 0, #fff;
    background-size: contain, auto, auto;
    position: fixed;
    top: 0;
    left: 0;
    display: flex;
    flex-direction: column;
    justify-content: center;
}

.loginForm{
    display: flex;
    flex-direction: column;
    justify-content: center;
    color: black;
    width: 620px;
    height:100vh;
    position: absolute;
    right: 90px;

}

.form{
    width: 356px;
    font-weight: 400;
}


h1{
    text-align: center;
}
