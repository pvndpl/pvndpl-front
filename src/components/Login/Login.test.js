import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Login from './Login';

describe('<Login />', () => {
  test('it should mount', () => {
    const element = render(<Login />);

    expect(element).toMatchSnapshot();

  });
});