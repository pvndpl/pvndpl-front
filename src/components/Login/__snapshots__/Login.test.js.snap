// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login /> it should mount 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="vkloginHeader"
      >
        <p
          class="vkloginHeaderPretitle"
        >
          Добро пожаловать в
        </p>
        <p
          class="vkloginHeaderTitle"
        >
          ПИВАНДОПАЛУ
        </p>
        <div
          class="loginForm"
        >
          <div
            class="container"
          >
            <div
              class="screen"
            >
              <div
                class="screen__content"
              >
                <h1>
                  Заходи
                </h1>
                <form
                  class="login"
                >
                  <div
                    class="login__field"
                  >
                    <input
                      class="login__input"
                      placeholder="User name"
                      type="text"
                    />
                  </div>
                  <div
                    class="login__field"
                  >
                    <input
                      class="login__input"
                      placeholder="Пароль"
                      type="password"
                    />
                  </div>
                  <button
                    class="button login__submit"
                  >
                    <span
                      class="button__text"
                    >
                      Зайти
                    </span>
                  </button>
                </form>
              </div>
              <div
                class="screen__background"
              >
                <span
                  class="screen__background__shape screen__background__shape4"
                />
                <span
                  class="screen__background__shape screen__background__shape3"
                />
                <span
                  class="screen__background__shape screen__background__shape2"
                />
                <span
                  class="screen__background__shape screen__background__shape1"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="vkloginHeader"
    >
      <p
        class="vkloginHeaderPretitle"
      >
        Добро пожаловать в
      </p>
      <p
        class="vkloginHeaderTitle"
      >
        ПИВАНДОПАЛУ
      </p>
      <div
        class="loginForm"
      >
        <div
          class="container"
        >
          <div
            class="screen"
          >
            <div
              class="screen__content"
            >
              <h1>
                Заходи
              </h1>
              <form
                class="login"
              >
                <div
                  class="login__field"
                >
                  <input
                    class="login__input"
                    placeholder="User name"
                    type="text"
                  />
                </div>
                <div
                  class="login__field"
                >
                  <input
                    class="login__input"
                    placeholder="Пароль"
                    type="password"
                  />
                </div>
                <button
                  class="button login__submit"
                >
                  <span
                    class="button__text"
                  >
                    Зайти
                  </span>
                </button>
              </form>
            </div>
            <div
              class="screen__background"
            >
              <span
                class="screen__background__shape screen__background__shape4"
              />
              <span
                class="screen__background__shape screen__background__shape3"
              />
              <span
                class="screen__background__shape screen__background__shape2"
              />
              <span
                class="screen__background__shape screen__background__shape1"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
